aircan/dependencies/google_cloud/bigquery_handler.py

```diff
@@ -31,9 +31,23 @@ def bq_import_csv(table_id, gcs_path, table_schema):
     
 def bq_schema_from_table_schema(table_schema):
     mapping = {
-        'number': 'float',
-        'year': 'integer'
-        }
+        'string': 'string',
+        "number": 'numeric',
+        "integer": 'numeric',
+        "boolean": 'boolean',
+        'object':'string',
+        "array":'string',
+        'date':'date',
+        'time':'time',
+        'datetime': 'datetime',
+        "year": 'numeric',
+        "yearmonth": 'string',
+        "duration":'datetime',
```

Comment:
@sagargg could you double check this one?

Reply:
@anuveyatsu ref. this documentation i think it is correct. 
https://cloud.google.com/bigquery/docs/schemas